### PR TITLE
Role management

### LINKED ISSUE #1251: Missing documentation for Administration -> Roles functionality
The relatively new account roles setup that was added over the past few years hasn't been documented, this means that some folks are promoting people to "owner" role, just to gain "view_devops" permissions. Ideally we would have documentation with screenshots of the roles UI, information about how to create new roles, and set their permissions.

### PR DESCRIPTION
**Pull request by liamstojanovic**
# Summary
A description of the roles and their management from the admin dashboard. Roles and their attributes are documented. Default roles present in a new server instance are identified and explained. Creation of custom roles, and editing existing roles are demonstrated with accompanying screenshots.

# Notes
- This page resides in the `~/content/en/admin` folder, which appears to be a rollup of installation, configuration, and maintenance of a server. It may be advantageous to organize the `admin` folder with high-level categorization, such as `Configuration/Installation` and `Maintenance`.
- I noticed other pages in the `admin` folder have `weight` properties. It was not immediately apparent what weight to assign my page.
- This page pertains specifically to managing roles through the web interface. 

# Related issue
fixes #1251

### CODE REVIEW

**Comment by andypiper** [comment]
This looks good (I'm just checking locally to confirm everything is working as expected).

I think we probably need to think more broadly about (re)organising the Admin section of the docs, particularly since this useful new addition currently lives at the bottom of a long and slightly out-of-sequence topics under the Admin index in the sidebar.

File: content/en/admin/roles.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
+---
+title: Roles
+description: Management of roles from the admin dashboard.
+menu:
+  docs:
+    parent: admin
+---
+
+# Roles {#roles}
+When the database is seeded, roles are derived from the values present in  [`~/config/roles.yml`](https://github.com/mastodon/mastodon/blob/main/config/roles.yml).
```
Comment (andypiper) [inline content/en/admin/roles.md:10]: trailing whitespace

**Review by andypiper — CHANGES_REQUESTED** [review]
`git apply` is reporting whitespace errors for 7 lines in the supplied new roles.md file.

**Comment by liamstojanovic** [comment]
> `git apply` is reporting whitespace errors for 7 lines in the supplied new roles.md file.

To the best of my knowledge, I have rectified trailing and double whitespace in the `roles.md` file.

It was not immediately clear to me how exactly to use the `git apply` command to output whitespace errors, so apologies if that error is still present.

**Review by vmstan — APPROVED** [review]

**Review by andypiper — APPROVED** [review]

**Pull request merged**